.rubocop.yml

```diff
@@ -48,6 +48,9 @@ Metrics/ModuleLength:
   Exclude:
     - 'spec/**/*' # not a big deal in spec helper module
 
+Style/DateTime:
```

Comment:
Just deleting the part that disables it is enough. Adding `Enabled: true` isn't necessary, as it's on by default.